android/app/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+plugins {
```

Comment:
this file already exist in the code base, why is it a newly added file on your PR ?